llvm/test/CodeGen/Hexagon/swp-ws-weak-dep.mir

```diff
@@ -0,0 +1,39 @@
+# REQUIRES: asserts
+# RUN: llc --march=hexagon %s -run-pass=pipeliner -debug-only=pipeliner \
+# RUN: -window-sched=force -o - 2>&1 | FileCheck %s
```

Comment:
Use -filetype=null since stdout is ignored

Reply:
Updated